dashboard/config/scripts/levels/Allthethings Java Lab 2.level

```diff
@@ -11,17 +11,14 @@
         "text": "import java.util.Scanner;\n\npublic class MyClass {\n  public static void main(String[] args) {\n    Scanner myScanner = new Scanner(System.in);\n    System.out.println(\"What's your name?\");\n    String response = myScanner.nextLine();\n    System.out.println(\"Hello \" + response + \"!\");\n  }\n}\n",
         "isVisible": true
       }
-    },
-    "encrypted_examples": [
-
-    ]
+    }
```

Comment:
Do you want these changes? Might not matter

Reply:
I was getting weird errors where it thought this level had validation so I cleaned up the configuration. May have been some conflict in my environment but it shouldn't impact anything to have this cleaned up.

Reply:
It looks like only change is removing blank `encrypted_examples` array? Would that tie to validation? Maybe not worth figuring out.

Reply:
yeah probably not, but it's not needed anyway so I think I'll leave as-is